functional_tests/README.md

```diff
@@ -0,0 +1,147 @@
+Docker Container Functional Tests
+========================================
+
+This code builds an image that serves as a framework to run functional tests on
+a target image. The image under tests runs as a docker container inside of this
+image, which itself runs as a docker container on a host machine in the cloud
+(when run through a
+[Google Cloud Container Build](https://cloud.google.com/container-builder/docs/overview)
+).
+
+# How Use This Image
+
+This image can be first built using the included `cloudbuild.yaml`. The
```

Comment:
It doesn't have to be project specific. I can publish it to gcr.io/gcp-runtimes. Maybe we can do that as a follow-up? Could you point me to a guide on what's involved in doing so?